### DISCUSSION TITLE
Failed to resolve component: NuxtImg

### DISCUSSION THREAD

**Discussion by simonbotte** [nuxt/image#750] [Q&A]
Hi 👋

I have this error when trying to use `<nuxt-img/>` : `Failed to resolve component: NuxtImg`

Here's a little summary of what I did : 
- `npm install -D @nuxt/image-edge` to my Nuxt3 project
- Edit my `nuxt.config.ts` module section to have this 
```ts 
modules: [
  "@nuxt/image-edge",
  "@pinia/nuxt"
],
```

- Replace a `<img/>` tag by `<NuxtImg/>` to have this 
``` vue
<NuxtImg width="90" height="90" :src="aType.imageUrl" :alt="'Close up on ' + aType?.name" />
```

- `aType.imageUrl` is a distant url so I added this in `nuxt.config.ts` 
```ts
image: {
  domains: ["image.domain.com"],
},
```

My issue here is that `<NuxtImg/>` isn't recognize. 

Do I did something wrong ?
Thank you for your appreciate help

**Comment by simonbotte** [comment]
I even try with `<nuxt-img/>` instead of `<NuxtImg/>` but nothing work

**Reply by Eng-khaledElsaeed** [reply]
try to execute this command in your terminal and it will be solved 

yarn add --dev @nuxt/image@rc
